lemminx-maven/src/test/resources/settings-local-repo-test-env-variable.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<settings xmlns="http://maven.apache.org/SETTINGS/1.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.0.0                       http://maven.apache.org/xsd/settings-1.0.0.xsd">
+    <localRepository>${env.MAVEN_PROJECTBASEDIR}/.m2/bug383repository</localRepository>
```

Comment:
This would rather be something like `${env.HOSTNAME}/.m2/bug383repository`

Reply:
If we do so then we'll be testing the same `${HOSTNAME}` value to create a local repo path in both - system property  and environment variable tests (I've already used it it system property test). But if it's OK I can change it to be `/${HOSTNAME}` for both test cases.